boards/we/orthosie1ev/we_orthosie1ev.dts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024 Würth Elektronik GmbH & Co. KG
```

Comment:
? this board does not seem to exist

Reply:
You are right it was added after the v4.0 was released! I'll remove it.